src/full/Agda/Interaction/BasicOps.hs

```diff
@@ -426,80 +425,82 @@ reifyElimToExpr = \case
+        p <- open p
+        t <- open t
+        pPi' "o" p (\ o -> t)
+  reify (ElimCmp cmp _ t v es1 es2) =
+    CmpElim cmp <$> reify t <*> mapM reifyElimToExpr es1
+                            <*> mapM reifyElimToExpr es2
+  reify (LevelCmp cmp t t')    = CmpLevels cmp <$> reify t <*> reify t'
+  reify (SortCmp cmp s s')     = CmpSorts cmp <$> reify s <*> reify s'
+  reify (UnquoteTactic tac _ goal) = do
+      tac <- A.App defaultAppInfo_ (A.Unquote exprNoRange) . defaultNamedArg <$> reify tac
+      OfType tac <$> reify goal
+  reify (UnBlock m) = do
+      mi <- lookupMetaInstantiation m
+      m' <- reify (MetaV m [])
+      case mi of
+        BlockedConst t -> do
+          e  <- reify t
+          return $ Assign m' e
+        PostponedTypeCheckingProblem cl -> enterClosure cl $ \case
+          CheckExpr cmp e a -> do
+              a  <- reify a
+              return $ TypedAssign m' e a
+          CheckLambda cmp (Arg ai (xs, mt)) body target -> do
+            domType <- maybe (return underscore) reify mt
+            target  <- reify target
+            let mkN (WithHiding h x) = setHiding h $ defaultNamedArg $ A.mkBinder_ x
+                bs = mkTBind noRange (fmap mkN xs) domType
+                e  = A.Lam Info.exprNoRange (DomainFull bs) body
+            return $ TypedAssign m' e target
+          CheckArgs _ _ _ args t0 t1 _ -> do
+            t0 <- reify t0
+            t1 <- reify t1
+            return $ PostponedCheckArgs m' (map (namedThing . unArg) args) t0 t1
+          CheckProjAppToKnownPrincipalArg cmp e _ _ _ t _ _ _ _ -> TypedAssign m' e <$> reify t
+          DoQuoteTerm cmp v t -> do
+            tm <- A.App defaultAppInfo_ (A.QuoteTerm exprNoRange) . defaultNamedArg <$> reify v
+            OfType tm <$> reify t
+        Open{}  -> __IMPOSSIBLE__
+        OpenInstance{}  -> __IMPOSSIBLE__
+        InstV{} -> __IMPOSSIBLE__
+  reify (FindInstance m mcands) = FindInstanceOF
+    <$> reify (MetaV m [])
+    <*> (reify =<< getMetaType m)
+    <*> forM (fromMaybe [] mcands) (\ (Candidate q tm ty _) -> do
+          (,,) <$> reify tm <*> reify tm <*> reify ty)
+  reify (IsEmpty r a) = IsEmptyType <$> reify a
+  reify (CheckSizeLtSat a) = SizeLtSat  <$> reify a
+  reify (CheckFunDef i q cs err) = do
+    a <- reify =<< defType <$> getConstInfo q
+    return $ PostponedCheckFunDef q a err
+  reify (HasBiggerSort a) = OfType <$> reify a <*> reify (UnivSort a)
+  reify (HasPTSRule a b) = do
+    (a,(x,b)) <- reify (unDom a,b)
+    return $ PTSInstance a b
+  reify (CheckDataSort q s) = DataSort q <$> reify s
+  reify (CheckLockedVars t _ lk _) = CheckLock <$> reify t <*> reify (unArg lk)
+  reify (CheckMetaInst m) = do
+    t <- jMetaType . mvJudgement <$> lookupLocalMeta m
+    OfType <$> reify (MetaV m []) <*> reify t
+  reify (CheckType t) = JustType <$> reify t
+  reify (UsableAtModality _ _ mod t) = UsableAtMod mod <$> reify t
+  {-# SPECIALIZE reify :: Constraint -> TCM (ReifiesTo Constraint) #-}
```

Comment:
There is a large diff just fixing some indentation.  Could whitespace changes be put into a separate commit please? (can then be ignored in a git blame).